draft-ietf-rats-architecture.md

```diff
@@ -1013,6 +1013,46 @@ an Attestation Result for any other purpose.
 
 # Security Considerations
 
+## Attester and Attestation Key Protection
+
+(this is a crude first pass to see what interest there is in this)
+
+Implementors need to pay close attention to the isolation and protection of the Attester and the factory processes for provisioning the Attestation Key Material. When these are compromised, the attestation becomes worthless because the attacker can forge attestations.
```

Comment:
```suggestion
Implementors need to pay close attention to the isolation and protection of the Attester and the factory processes for provisioning the Attestation Key Material. When these factory processes are compromised, the attestation becomes worthless because the attacker can forge attestations.
```